octomap_to_gridmap_demo:
  ros__parameters:
    octomap_service_topic: "/octomap_binary"
    min_z: -2.0
    max_z: 2.0

grid_map_visualization:
  ros__parameters:
    grid_map_topic: /grid_map
    grid_map_visualizations: [elevation_points, elevation_grid, map_region]
    elevation_points:
      type: point_cloud
      params:
        layer: elevation
    elevation_grid:
      type: occupancy_grid
      params:
        layer: elevation
        data_min: -2.0
        data_max: 2.0
    map_region:
      type: map_region
      params:
        color: 3289650
        line_width: 0.1
